diagrams/metametamodel-gen.puml

```diff
@@ -50,23 +45,23 @@ abstract class NamespacedEntity {
   qualifiedName(): String
 }
 
-class PrimitiveType extends Datatype
+class PrimitiveType extends DataType
 
 class Property extends Feature
 
 class Reference extends Link
 
 ' primitive type: "String"
 
-class Typedef extends Datatype
+class Typedef extends DataType
 
 ' primitive type: "boolean"
```

Comment:
Also this comment is not clear

Reply:
Same thing as above: a consequence of a primitive type def. in the self-definition of M3/lioncore. You can ignore it.